dwh/indicators.py

```diff
@@ -84,6 +84,22 @@ def create_args(parser: argparse.ArgumentParser):
       type=str
   )
 
+  parser.add_argument(
+      '--location_ids',
+      help='A list of location Ids that should be kept',
+      nargs='+',
```

Comment:
Should this and the next `nargs` be `'*'` instead of `'+'`?

Reply:
when using '+' requires at least one argument, while * can allow 0 argument. I think + is desirable

Reply:
Oh, right, the default is `[]`.